plothist/plotters.py

```diff
@@ -76,6 +132,33 @@ def update_variable_registry(
     variable_key, x_min, x_max, path="./variable_registry.yaml"
 ):
     # TODO: extend updating function
+    """
+    Update the range parameter for a variable in the variable registry file.
+
+    Parameters
+    ----------
+    variable_key : str
+        The key identifier of the variable to update in the registry.
+    x_min : float
+        The new minimum value for the range of the variable.
+    x_max : float
+        The new maximum value for the range of the variable.
+    path : str, optional
+        The path to the variable registry file (default is "./variable_registry.yaml").
+
+    Returns
+    -------
+    None
+
+    Raises
+    ------
+    RuntimeError
+        If the variable registry file does not exist, and the default path is used.
```

Comment:
weird

Reply:
The idea is:

If the file does not exist, 2 scenarios:

1.The case specified in the documentation: the user forgot to run `create_var_reg()`, the `RuntimeError` (here in the doc) tells him to run this function
2. the user inputted an incorrect `yaml` path (so he knows what he do), so the `safe_load()` of the `yaml` returns a `RuntimeError` incorrect path

The documentation might be misleading, I thing removing this `RuntimeError` from the documentation is better.